### PR TITLE
Remove deployments on turing

### PR DESCRIPTION
**Pull request by jchristgit**
Files of the webserver have been migrated over.

### CODE REVIEW

**Review by jb3 — COMMENTED** [review]
Should we consider removing the now-redundant wireguard role?

**Review by shtlrs — APPROVED** [review]

File: ansible/local_testing/README.md [hunk 1]
```diff
@@ -55,7 +55,6 @@ vms:
 - lovelace: 192.168.56.3
 - neumann: 192.168.56.4
 - richie: 192.168.56.5
-- turing: 192.168.56.6
```
Comment (jb3) [inline ansible/local_testing/README.md:58]: This is probably a good time to update the whole group here to reflect the new removed hosts.

**Review by jb3 — APPROVED** [review]
Looks solid! One minor thing if you feel like changing it.

**Review by jb3 — APPROVED** [review]

**Pull request merged**